neutron_lbaas_dashboard/static/dashboard/project/lbaasv2/listeners/actions/edit/wizard.controller.js

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
Do we really need this empty line?

Reply:
It's optional, eslint pays no attention to it.

To my opinion, the license comment and 'function() {' are 2 differnent parts. The empty line can be a separate mark for them.

Reply:
Personally, I will prefer to upstream such kind of change to community, instead of dropping them to our repo, in order to reduce the possibility of rebasing conflict..

Reply:
done to remove the empty line.